keyboards/handwired/scottokeebs/scotto34/info.json

```diff
@@ -0,0 +1,77 @@
+{
+    "manufacturer": "ScottoKeebs",
+    "keyboard_name": "Scotto34 (PCB Edition)",
+    "maintainer": "joe-scotto",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": [
+            "GP9",
+            "GP10",
+            "GP11",
+            "GP12",
+            "GP13",
+            "GP18",
+            "GP19",
+            "GP20",
+            "GP21",
+            "GP22"
+        ],
+        "rows": ["GP0", "GP1", "GP14", "GP15"]
+    },
+    "processor": "RP2040",
+    "url": "https://scottokeebs.com/products/scotto34-pcb-kit",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x3334",
+        "vid": "0x534B"
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
This looks like it matches split_3x5_2?

Reply:
Nope, bottom row has 5 keys... w left, middle space bar, then 2 right of that.

Reply:
@drashna Nope, bottom row has 5 keys... 2 left, middle space bar, then 2 right of that.

Reply:
@drashna Whoops, the Scotto34 directory shouldn't have been in there as it's in other PRs. I removed it in [3918eac](https://github.com/qmk/qmk_firmware/pull/20937/commits/3918eaca1b1ae83d99c27fe28b16800ce4a98a8e).